infra/us-east-1/prod/rds/bastion/security-group/terragrunt.hcl

```diff
@@ -0,0 +1,40 @@
+locals {
+
+  # Automatically load environment-level variables
+  account_vars     = read_terragrunt_config(find_in_parent_folders("account.hcl"))
+
+  # Extract out common variables for reuse
+  project        = local.account_vars.locals.project
+  app            = "marlowe-runtime"
+
+  bastion_name          = "${local.project}-${local.app}-bastion"
+  bastion_key_pair_name = "${local.project}-${local.app}-db-bastion-kp"
+
+  tags = {
+    Owner        = "Terraform"
+    tribe        = "smart-contracts"
+    App          = "marlowe-runtime"
+  }
+
+}
+
+terraform {
+  source = "github.com/terraform-aws-modules/terraform-aws-security-group//.?ref=v5.1.0/modules/ssh"
+}
+
+# Include all settings from the root terragrunt.hcl file
+include {
+  path = find_in_parent_folders()
+}
+
+dependency "vpc" {
+  config_path = "../../../vpc"
+}
+
+inputs = {
+  vpc_id = dependency.vpc.outputs.vpc_id
+
+  computed_ingress_cidr_blocks = ["0.0.0.0/0"]
+  ingress_cidr_blocks          = ["0.0.0.0/0"]
```

Comment:
As you're already setting VPC as a dependency, why not using the outputs to declare these CIDRs?

```hcl
inputs = {
  name                = local.security_group_name
  vpc_id              = dependency.vpc.outputs.vpc_id
  ingress_cidr_blocks = dependency.vpc.outputs.private_subnets_cidr_blocks
}
```

Reply:
These are supposed to be the CIDRs for allowed IPs in the security group. Those can't be extracted from the VPC output.